### PR TITLE
Master

### PR DESCRIPTION
**Pull request by DeVerMyst**
Ready for release
Contains 3 demo: old one, gravity bc16 and giant monochromatic bc04.
Possibility to initialise PAINTER automatically with keyword `init =
true` : alpha, beta rho_y_xi and rho_y_gamma are automaticaly setted.

Data are normalized by global flux which allow to Use in most part of
the case ready parameters.

Doc is updated

### CODE REVIEW

File: src/paintertools.jl [hunk 1]
```diff
@@ -428,30 +406,33 @@
...
   ## PAINTER Matrices creation, Array Initialization
     println("")
-    PDATA,OIDATA  = painterarrayinit(PDATA, OIDATA)
+    PDATA,OIDATA = painterarrayinit(PDATA, OIDATA)
   # Check, Create PAINTER object and mask initialisation from data or fits
-    OIDATA.mask3D   = checkmask(OIDATA.mask3D, OIDATA.nx, OIDATA.nw)
+    OIDATA.mask3D = checkmask(OIDATA.mask3D, OIDATA.nx, OIDATA.nw)
   # Initialise Data and xinit from V2 and flux
-    PDATA,OIDATA = painterautoparametersinit(PDATA,OIDATA,flux)
+    if init == "autoinit"
```
Comment (andferrari) [inline src/paintertools.jl]: @DeVerMyst should be  `if autoinit == "auto"`

**Review by andferrari — COMMENTED** [review]
bug to correct

File: src/paintertools.jl (hunk 1 above)
Comment (DeVerMyst) [reply src/paintertools.jl]: Yes totally ! 
last minute modification :)

Comment (andferrari) [reply src/paintertools.jl]: c'est à toi de faire la modif, moi je ne peux pas !

**Review by andferrari — CHANGES_REQUESTED** [review]

**Comment by DeVerMyst** [comment]
Ah ok, je pensais que tu avais la main dessus.
C'est fait !

2016-10-24 20:05 GMT+02:00 André Ferrari notifications@github.com:

**Pull request merged**